internal/migration_acceptance_tests/partitioned_index_cases_test.go

```diff
@@ -623,6 +618,164 @@ var partitionedIndexAcceptanceTestCases = []acceptanceTestCase{
 			`,
 		},
 	},
+	{
+		name: "Add primary key constraint with existing matching base-table index (matching child index does not exist)",
+		oldSchemaDDL: []string{
+			`
+			CREATE TABLE foobar(
+			    id INT NOT NULL,
+				foo VARCHAR(255) NOT NULL
```

Comment:
is this indent for foo correct?

Reply:
Looks like the entire file is messed up. It renders okay in my editor...I'm guessing this is using tabs instead of spaces, which happens to align okay in my editor. I'll fix in a subsequent PR